app/admin.go

```diff
@@ -174,6 +175,20 @@ func (a *App) RecycleDatabaseConnection() {
 	mlog.Warn("Finished recycling the database connection.")
 }
 
+func (a *App) TestSiteURL(userId string, data *model.TestSiteURL) *model.AppError {
```

Comment:
`userId` is not used here.  I suggests:
```suggestion
func (a *App) TestSiteURL(siteURL string) *model.AppError {
```